setup.py

```diff
@@ -56,6 +56,7 @@
         "eth-rlp>=0.1.2,<1",
         "eth-utils>=1.3.0,<2",
         "hexbytes>=0.1.0,<1",
+        "py-hdwallet>=0.1.0a1",
```

Comment:
BTW, we'd typically want to at least put `py-hdwallet` into beta before releasing this.